internal/sirius/permissions.go

```diff
@@ -0,0 +1,45 @@
+package sirius
+
+import (
+	"context"
+	"encoding/json"
+	"errors"
+	"net/http"
+	"strconv"
+)
+
+type PermissionGroup struct {
+	Permissions []string `json:"permissions"`
+}
+
+type PermissionSet map[string]PermissionGroup
+
+type MyPermissions struct {
```

Comment:
This isn't used outside of here so can be renamed to `myPermissions` to make it private